zgrab2_schemas/zgrab2/ipp.py

```diff
@@ -147,15 +147,56 @@
     "request": http_request_full
 })
 
+# TODO: Determine whether value-tag types with same underlying form should have a different name in this mapping
+# TODO: Add method to decode these values appropriately. Encoding of different tag's attribute values specified
+# in Table 7 of RFC 8010 Section 3.9 (https://tools.ietf.org/html/rfc8010#section-3.9)
+# "value-tag" values which specify the interpretation of an attribute's value:
+# From RFC 8010 Section 3.5.2 (https://tools.ietf.org/html/rfc8010#section-3.5.2)
+# Note: value-tag values are camelCase because the names are specified that way in RFC
+ipp_attribute_value = SubRecord({
+    "raw": Binary(),
```

Comment:
At some point, it would be nice to give all of these `doc` (and maybe `example`) tags (see e.g. https://github.com/zmap/zgrab2/blob/master/zgrab2_schemas/zgrab2/mysql.py#L62).